test/e2e/e2e.bats

```diff
@@ -6,3 +6,21 @@ source test/e2e/helpers.sh
 	result="$(shp --help)"
 	[ ! -z "$result" ]
 }
+
+@test "shp build create succesfully run with 1 argument" {
+	build=$(randomDnsSubdomainName)
+	result="$(shp build create ${build} --source-url=url --output-image=image)"
+        [ ! -z "$result" ]
+	result=$(kubectl get build.shipwright.io ${build})
+	[ ! -z "$result" ]
+}
+
+@test "shp buildrun create succesfully run with 1 argument" {
+        build=$(randomDnsSubdomainName)
+        result="$(shp build create ${build} --source-url=url --output-image=image)"
+        [ ! -z "$result" ]
+	buildrun=$(randomDnsSubdomainName)
```

Comment:
I recommend using the same convention than in Bash scripts, so the name would be `build_run` instead.